src/components/Carousel.js

```diff
@@ -225,12 +225,12 @@ class Carousel extends Component {
         const { axis } = this.props;
         const isHorizontal = axis === 'horizontal';
 
-        const nextKey = isHorizontal ? 'ArrowRight' : 'ArrowDown';
-        const prevKey = isHorizontal ? 'ArrowLeft' : 'ArrowUp';
+        const nextKey = isHorizontal ? 39 : 40;
```

Comment:
Could you create a map to identify these numbers with a readable name? 

example:
```
const keyNames = {
    ArrowRight: 39,
    ArrowDown: 40 
};

const nextKey = isHorizontal ? keyNames.ArrowRight : keyNames.ArrowDown;
```